scripts/tests/linux-group.sh

```diff
@@ -11,32 +11,32 @@ SERVICE_NAME=firezone-client-ipc
 SOCKET=/run/dev.firezone.client/ipc.sock
 export RUST_LOG=info
 
+cd rust || exit 1
+cargo build --bin "$BINARY_NAME"
+cd ..
+
+function debug_exit() {
+    systemctl status "$SERVICE_NAME"
+    exit 1
+}
+
 # Copy the Linux Client out of the build dir
-sudo cp "rust/target/debug/firezone-headless-client" "/usr/bin/$BINARY_NAME"
+sudo cp "rust/target/debug/$BINARY_NAME" "/usr/bin/$BINARY_NAME"
 
 # Set up the systemd service
 sudo cp "rust/gui-client/src-tauri/deb_files/$SERVICE_NAME.service" /usr/lib/systemd/system/
 sudo cp "scripts/tests/systemd/env" "/etc/default/firezone-client-ipc"
 
 # The firezone group must exist before the daemon starts
 sudo groupadd "$FZ_GROUP"
-sudo systemctl start "$SERVICE_NAME" || { systemctl status "$SERVICE_NAME"; exit 1; }
+sudo systemctl start "$SERVICE_NAME" || debug_exit
 
 # Make sure the socket has the right permissions
 if [ "root $FZ_GROUP" != "$(stat -c '%U %G' $SOCKET)" ]
 then
     exit 1
 fi
 
-# Add ourselves to the firezone group
-sudo gpasswd --add "$USER" "$FZ_GROUP"
-
-echo "# Expect Firezone to accept our commands if we run with 'su --login'"
-sudo su --login "$USER" --command RUST_LOG="$RUST_LOG" "$BINARY_NAME" stub-ipc-client
-
-echo "# Expect Firezone to reject our command if we run without 'su --login'"
-"$BINARY_NAME" stub-ipc-client && exit 1
```

Comment:
`stub-ipc-client` was already removed from the code, so I'm not sure why this test is still passing on main. I'm removing this part because it's invalid and it's not needed, if the user and group ownership are correct we know it will work.

Reply:
Uh oh, maybe the test is a false positive?

Reply:
Yeah. I'm not sure why, because there's one positive check and one negative check. If a subcommand doesn't exist, clap should cause us to return non-zero every time. 

It might be that the subcommand was interpreted as a token, but then it should still have a consistent return code.